CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0 (2020-07-28)
+
+- Added a directive `skyAgGridDataManagerAdapter` to handle interaction between ag-Grid and SKY UX data manager. [#59](https://github.com/blackbaud/skyux-ag-grid/pull/59)
```

Comment:
I think "a directive `skyAgGridDataManagerAdapter`" should be `the `skyAgGridDataManagerAdapter` directive`.

Also, shouldn't "ag-Grid" be "the data entry grid component" since that's how we refer to the component in the SKY UX docs? And should "SKY UX data manager" be "the data manager component"?